cloudfront/wellcomelibrary.org/terraform/cf_behaviours.tf

```diff
@@ -118,6 +118,38 @@ locals {
   ]
 
   static_redirects_behaviours_stage = [
+    {
```

Comment:
This list is not yet exhasutive.

Reply:
Should we have these as a list to map over?

Reply:
Relatively soon this will just be everything, so i'm not sure how much value there is in optimising it just to remove it again next week.